applications/web3-open-jurisdiction-stack.md

```diff
@@ -0,0 +1,346 @@
+
+We also aim to release at least one open source component of the stack to engage and share with the community around how the development of the jurisdiction stack could look like. This endeavour will be done also in case of “no-go” outcome from the research. Sentiment and needs towards a jurisdiction may change in the future and we think it would be wise at least to leave a tangible outcome from this endeavour from where anyone can resume.
+
+**This is a place for you to manage expectations and clarify any limitations that might not be obvious**
+
+To envision and establish an open jurisdiction is a demanding undertaking that necessitates iterative processes and real-world usage, much like the evolution of legal systems within tribes, communities, and modern nation-states.
+
+We recognize that the actual development of such a jurisdiction, considering the unique nuances of web3, will require multiple grants and a collaborative effort from the community. As part of our research, we aim to outline a possible architecture that the community can strive towards, shedding light on the key components and design considerations necessary for its realisation.
+
+We will push the bar by releasing at least one example pallet (or SC) on how one component of the jurisdiction stack could be implemented. In this way, in the future, anyone could leverage the research, architecture and POC to develop a production ready software.
+
+We also would like to clarify that this activity does not directly involve anything specific to the Jur project where most of our team is involved. Jur is a L1 solochain that powers the creation of societies and Network States. There is an overlap between the two and this is why our expertise is relevant but the goal of this project is to build an open source stack that can serve the entire ecosystem (including Jur). Given its goal it is highly likely that the research will suggest the stack to take the shape of a system parachain that in no way is related to Jur directly.
+
+Disclaimer: Jur has built an arbitration platform, the “Open Justice Platform”, that supports legally binding resolutions in 160+ jurisdictions. This platform could be one of the resolution methods available in the stack, alongside all the others that the community will see fit. In no way, this will be the suggested or required solution. It will be on the parties involved to choose the best resolution methods for their needs.
+
+### Ecosystem Fit
+
+
+**Help us locate your project in the Polkadot/Substrate/Kusama landscape and what problems it tries to solve by answering each of these questions:**
+
+**Where and how does your project fit into the ecosystem?**
+
+Having actively participated in the Dotsama ecosystem over the past year and engaging with numerous stakeholders, we have gained valuable insights into the needs inherent to the ecosystem.
```

Comment:
Could you give some specific examples, such as discussions/threads/blog articles/podcasts/conferences you've contributed to?